build.Dockerfile

```diff
@@ -12,7 +12,8 @@ RUN go get github.com/jteeuwen/go-bindata/... \
 && go get github.com/kisielk/errcheck \
 && go get github.com/client9/misspell/cmd/misspell \
 && go get github.com/mattfarina/glide-hash \
-&& go get github.com/sgotti/glide-vc
+&& go get github.com/sgotti/glide-vc \
+&& go get github.com/Masterminds/sprig
```

Comment:
Please remove this, we don't manage dependencies like that: https://github.com/containous/traefik/blob/master/.github/CONTRIBUTING.md#setting-up-glide-and-glide-vc-for-dependency-management